projects/optic/src/commands/login/login.ts

```diff
@@ -56,24 +61,43 @@ Create an account and generate a personal access token at ${chalk.underline.blue
     { onCancel: () => process.exit(1) }
   );
 
-  if (response.token) {
-    const base64Token = Buffer.from(response.token).toString('base64');
+  if (!response.token) {
+    throw new Error('Expected token');
+  }
 
-    const newConfig = userConfig
-      ? {
-          ...userConfig,
-          token: base64Token,
-        }
-      : {
-          token: base64Token,
-        };
-    await fs.mkdir(path.dirname(USER_CONFIG_PATH), { recursive: true });
-    await fs.writeFile(USER_CONFIG_PATH, JSON.stringify(newConfig), 'utf-8');
+  logger.info(chalk.green(`\nVerifying your token`));
+  const newClient = createOpticClient(response.token);
+  try {
+    const result = await newClient.verifyToken();
 
-    logger.info(
-      chalk.green(
-        `Successfully saved your personal access token to ${USER_CONFIG_PATH}`
-      )
-    );
+    if (result.user) {
+      identify(result.user.userId, result.user.email);
```

Comment:
We'll want to send the https://github.com/opticdev/optic/blob/main/projects/openapi-utilities/src/utilities/segment.ts#L8 as the user id instead which I think will help associate events with the email.

Alternatively I think there's an `alias` command that might be appropriate here

Reply:
Yeah I think that should work - we'll just want to see the result of that